cpp/CMakeLists.txt

```diff
@@ -84,6 +84,22 @@ endif()
 add_library(kvikio INTERFACE)
 add_library(kvikio::kvikio ALIAS kvikio)
 
+# Enable supported cuFile features in KvikIO
```

Comment:
I think it would be helpful to provide some comments on how this section differs from the final code block below. It makes sense to me right now, but it will be hard to remember the intention of each piece of code later on.

Reply:
I've added a thorough explanation.